docs/conf.py

```diff
@@ -1,9 +1,5 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
 import sys
 import os
-
```

Comment:
we need to keep this line to be PEP 8 compliant :) see https://www.python.org/dev/peps/pep-0008/#imports